src/Controls/src/Core/HandlerImpl/Button/Button.iOS.cs

```diff
@@ -9,10 +9,8 @@ namespace Microsoft.Maui.Controls
 {
 	public partial class Button
 	{
-		private static void MapPadding(IButtonHandler handler, Button button)
```

Comment:
Why are all the padding ones private?

Reply:
So we can avoid this type of PR for NET8 :-)